apps/allocations/app/components/App/BudgetDetail.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Redirect, useParams } from 'react-router-dom'
+import { useAragonApi } from '../../api-react'
+
+export default function BudgetDetail() {
+  const { appState } = useAragonApi()
+  const { id } = useParams()
+  const budget = appState.budgets.find(b => b.id === id)
+
+  if (!budget) {
+    return <Redirect to="/" />
+  }
+
+  return JSON.stringify(budget)
```

Comment:
This is a placeholder.

I want to get the routing structure merged, since lots of stuff in [the Allocations epic](https://github.com/AutarkLabs/product/issues/6) will conflict with this new structure, if work is started before this gets merged.

I will continue work on the BudgetDetail component once this gets merged.